api/HTMLMarqueeElement.json

```diff
@@ -124,10 +124,10 @@
               "version_added": "10.1"
             },
             "safari": {
-              "version_added": "3"
+              "version_added": false
```

Comment:
Ah, another miscapitalization. This is `bgcolor`, should be `bgColor`.

@vinyldarkscratch can you list all entries which are identical except for capitalization, and add this and any additional ones which are actually a typo in BCD to https://github.com/foolip/mdn-bcd-collector/pull/807?